WalletWasabi.Tests/UnitTests/Transactions/TransactionFactoryTests.cs

```diff
@@ -296,7 +296,8 @@ public void CustomChangeScript()
 			Assert.Equal(2, tx.Outputs.Count);
 
 			var changeOutput = Assert.Single<TxOut>(tx.Outputs, x => x.ScriptPubKey == changeKey.ScriptPubKey);
-			Assert.Null(transactionFactory.KeyManager.GetKeyForScriptPubKey(changeOutput.ScriptPubKey));
+			transactionFactory.KeyManager.TryGetKeyForScriptPubKey(changeOutput.ScriptPubKey, out HdPubKey? hdPubKey);
+			Assert.Null(hdPubKey);
```

Comment:
We don't care if it's null or not. We care about if it's true or not.

Reply:
Done